.gitignore

```diff
@@ -6,7 +6,7 @@ yarn-debug.log*
 yarn-error.log*
 pnpm-debug.log*
 lerna-debug.log*
-
+coverage/
```

Comment:
Same issue as the last time. Rebase correctly and you won't have to touch this file.